SoSe19/bericht_data.tex

```diff
@@ -0,0 +1,80 @@
+Das BAföG Bündnis hat hierzu ein Positionspapier \footnote{\url{https://www.bmbf.de/files/Stellungnahme\%20BAf\%c3\%b6G-B\%c3\%bcndnis.pdf}} verfasst, welchen die ZaPF unterstützt.
+		
+\section*{Akkreditierungsrichtlinien der ZaPF (kommentierte MRVO)}
+
+2018 haben sich im deutschen Akkreditierungssystem Änderungen ergeben, 
+zu denen die ZaPF sich bereits geäußert hat. 
+Auf der diesjährigen ZaPF wurde nun nach Vorbereitungen im letzten Jahr die Musterrechtsverordnung (MRVO) kommentiert, 
+sodass die Akkreditierungsrichtlinien der ZaPF Berücksichtigung finden, um somit den von der ZaPF entsendeten Gutachter*innen die Arbeit zu erleichtern.
+
+Zusätzlich soll eine kondensierte Version als Empfehlung für die Akkreditierungsagentur ASIIN entworfen werden, da diese ihre Richtlinien für die Akkreditierung von Physikstudiengängen gerade überarbeitet. 
+
+\section*{Beschäftigung mit aktuellen Entwicklungen im Wissenschaftssystem}
+
+Die ZaPF hat sich mit aktuellen Entwicklungen im Wissenschaftssystem auseinandergesetzt, darunter der Einführung der Nationalen Forschungsdateninfrastruktur (NFDI \footnote{\url{https://www.dfg.de/foerderung/programme/nfdi/}}) und Themen rund um Open Science. Zur NFDI wurde eine Referentin von der DFG eingeladen, die die Initiative vorstellte und anschließend mit den Teilnehmenden über die Ausgestaltung der NFDI und insbesondere ihre Rolle im Physikstudium diskutierte.
+Schwerpunkt in der Diskussion um Open Science waren die Themen Plan S \footnote{\url{https://www.coalition-s.org/about/}}, Project DEAL \footnote{\url{https://www.projekt-deal.de/}} und wie Open Science in der universitären Lehre verankert werden könnte.
+
+\section*{Bachelor-Master-Umfrage}
+
+Im Sommersemester 2018 wurde durch die ZaPF eine Umfrage unter den Physikstudierenden im deutschsprachigen Raum durchgeführt, welche zum Ziel hat, die Entwicklung des Physikstudiums in regelmäßigen Abständen zu beobachten. 
+Die Umfrage wurde zum dritten Mal durchgeführt und es nahmen 2750 Studierende aus dem Bachelor, Master oder Promotion daran teil. Es wurden hierbei allgemeine Fragen zu Bereichen wie Studienbelastung, subjektiver Wahrnehmung der Vermittlung von Qualifikationen und Kompetenzen, Vorbereitung auf das Studium sowie der Weg nach dem Studium gestellt. Darüber hinaus wurde analysiert, inwieweit die Themenkomplexe Ethik und Wissenschaftstheorie im Studium eingebunden werden sollten.
+
+Auf der ZaPF in Bonn hat sich ein Arbeitskreis darüber ausgetauscht, wie die Ergebnisse der Umfrage den Universitäten aufbereitet zur Verfügung gestellt werden können und an welchen Stellen sie außerdem eingesetzt werden können.
```

Comment:
Gab es dazu auch Erkenntnisse, die man hier benennen könnte? Ist geplant, die Ergebnisse in irgendeiner Weise zu veröffentlichen? Wenn ja, wäre hier doch ein richtiger Ort, um dies kundzutun.